pkg/relay/ethereum/events.go

```diff
@@ -160,6 +163,9 @@ func (chain *Chain) findRecvPacketEvents(ctx core.QueryContext, fromHeight uint6
 		if err != nil {
 			return nil, fmt.Errorf("failed to parse RecvPacket event: err=%v, log=%v", err, log)
 		}
+		if event.Packet.SourceChannel != chain.Path().ChannelID {
```

Comment:
In `findWriteAckEvents` you chose `DestinationChannel`, but you chose `SourceChannel` here. Why?

Reply:
Fix to `DestinationChannel`